### PR TITLE
Add documentation for Slack channel

### PR DESCRIPTION
**Pull request by Rafiramadhan**
### **_

> Thank you~

_**

### CODE REVIEW

**Comment by Rafiramadhan** [comment]
> App_credential image sensornya tolong dibenarkan

sudah ya kash, coba di cek kembali 🙏

File: docs/channels/slack.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+3. From "Basic Information" in App Credentials section you can see Verification Token, we will use it Later to deploy our bot.
+
+![Verification Token](./images/slack/App_credential.png)
+
+4. Open "OAuth & Permissions" and in "Scopes" section you should fill the scope permission by adding below scopes.
+
+![Filling Scopes](./images/slack/scopes_fill_fix.png)
+
+5. Open "Bot User" in Features from the sidebar menu. Fill the "Display name" and "Default username" of your bot. don't forget to set "Always Show My Bot as Online" to "On" and press "Add Bot User" button to finish.
```
Comment (indraaristya) [inline docs/channels/slack.md]: `don’t forget....` to `Don't forget....`

File: docs/channels/slack.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+![Bot User](./images/slack/bot_user.png)
+
+6. Go back to "Oauth & Permissions" and Select "Install App to Workspace" then click "Allow".
```
Comment (indraaristya) [inline docs/channels/slack.md]: `Oauth....` to `OAuth` and `Select` -> `select`

File: docs/channels/slack.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+![Install App to Workspace](./images/slack/install_app.png)
+![request_access](./images/slack//request_access.png)
+
+7. Still in the "Oauth & Permissions" you will get "OAuth Access Token" and "Bot User OAuth Access Token". we will use "Bot User OAuth Access Token" to deploy our bot using Kata Platform.
+
+## Create Slack Channnel in Kata Platform
```
Comment (indraaristya) [inline docs/channels/slack.md]: only double `n` in `Channnel` hehe

File: docs/channels/slack.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+1. Click "Start here" then "Create a Slack app"
+
+![Start here](./images/slack/start.png)
+![Create Slack Apps](./images/slack/create-slack-app.png)
+
+2. Fill "App Name". choose your "Development Slack Team" and click "Create App"
+
+![App Name](./images/slack/pop_up.png)
+
+3. From "Basic Information" in App Credentials section you can see Verification Token, we will use it Later to deploy our bot.
```
Comment (septiandrd) [inline docs/channels/slack.md]: later

File: docs/channels/slack.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+Slack is where work flows. It's where the people you need, the information you share, and the tools you use come together to get things done. In this section, we will learn to connect our bot with Telegram. You can use the bot you created in the [Hello World tutorial](/tutorial/hello-world/) to start integrating your bot with Telegram.
+
+## Create Slack Channel
+
+First thing first, you need to [sign in](https://slack.com/signin) into your Slack Team to create a new
+app and get its token.
+After logged in, open this [URL:](https://api.slack.com/slack-apps)
+
+1. Click "Start here" then "Create a Slack app"
```
Comment (septiandrd) [inline docs/channels/slack.md]: titik (end of sentence)

File: docs/channels/slack.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+![Start here](./images/slack/start.png)
+![Create Slack Apps](./images/slack/create-slack-app.png)
+
+2. Fill "App Name". choose your "Development Slack Team" and click "Create App"
```
Comment (septiandrd) [inline docs/channels/slack.md]: - `choose` > `Choose`
- titik

File: docs/channels/slack.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+![Fill Token](./images/slack/bottoken.png)
+![Create Token](./images/slack/createbot.png)
+
+## Setup Webhook URL in Slack Bot
+
+Once finished, you will see the newly-created channel in the list. You will also see a Webhook URL for your channel. This webhook URL will be added to the Slack bot. Copy the webhook URL by clicking "View Channel" and clicking the copy button next to the Webhook URL field.
+
+![Copy Webhook](./images/slack/copy_webhoook.png)
+
+Next, back to the browser, and go to "Event Subscriptions" tab then switch "Enable Events" to "on"
```
Comment (septiandrd) [inline docs/channels/slack.md]: titik

File: docs/channels/slack.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+![Event Subcriptions](./images/slack/event.png)
+![Enable Events](./images/slack/toggle_on.png)
+
+You will see "Request URL" field. Then put the webhook URL that previously copied here.
+
+![Request URL](./images/slack/setwebhook.png)
+
+Expand the "Subscribe to bot events" then add selected User Event like in the below image
```
Comment (septiandrd) [inline docs/channels/slack.md]: titik

File: docs/channels/slack.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+![Subscribe bot events](./images/slack/final.png)
+
+Now you can search for your bot in your slack Workspace
```
Comment (septiandrd) [inline docs/channels/slack.md]: titik

**Pull request merged**